apps/api_web/mix.exs

```diff
@@ -58,7 +58,7 @@ defmodule ApiWeb.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:phoenix, "< 1.4.17"},
+      {:phoenix, "< 1.5.2"},
```

Comment:
Should we add `{:phoenix_pubsub, "~> 2.0"}` to this list, per the [upgrade guide](https://gist.github.com/chrismccord/e53e79ef8b34adf5d8122a47db44d22f)? I see in `mix.lock` that it _was_ updated so it's more a matter of whether it's important to explicitly express the version constraint here.

Reply:
Nevermind. I see that phoenix itself has that constraint, so we're probably fine. I guess the guide is just being explicit about it in case we already had it in here for some reason.